test/python/modules/linear_boltzmann_solvers/transport_keigen/keigenvalue_transport_2d_1c_qblock_cbc.py

```diff
@@ -66,3 +66,6 @@
     )
     k_solver.Initialize()
     k_solver.Execute()
+
+    k = k_solver.GetEigenvalue()
+    print("k-eigenvalue =", k)
```

Comment:
Do you not want the test harness to check that the value was actually produced?

Reply:
the test relies on the c++ printout. do you suggest that i change the test to rely on the python + getter ?

Reply:
done